services/datamanager/data/fake_robot_with_data_manager.json

```diff
@@ -15,7 +15,7 @@
                         "capture_methods": [
                             {
                                 "method": "EndPosition",
-                                "capture_frequency_hz": 100,
+                                "capture_frequency_hz": 500,
```

Comment:
What is this change based off of? Just wondering

Reply:
Just further reducing the risk of flakiness by increasing the rate at which data is captured. It shouldn't _really_ make a difference though (basically just adds/removes an 8ms window where the capture routine may have started but is waiting for the first tick), so I changed it back

Reply:
I was wrong! This does totally matter in the context of this test, because we use buffered io. So since we're chcecking the thing on disk as it's being written (as opposed to only after closing the service/the underlying collectors/the underlying buffered writers), `waitUntil` will wait until the first flush occurs, which is when 4096 bytes have been written, so is a direct function of this. The higher this is the faster it will happen, so bumped this to 500.